File: JavaScript/8js 3/templeStrings.js
/*
|--------------------------------------------------------------------------
| Template Strings
|--------------------------------------------------------------------------
|
*/

function TempleStrings(obj, feature) {
   return `${obj} are ${feature}`;
}


const TempleStrings = (obj, feature) => `${obj} are ${feature}`;


const TempleStrings = function (obj, feature) {
   return obj + " are " + feature;
};


console.log(TempleStrings("Animals", "Good"), "Animals are Good");

File: Python/8py 3/temple_strings.py
"""
|--------------------------------------------------------------------------
| Template Strings
|--------------------------------------------------------------------------
|
"""

def temple_strings(obj, feature): 
   return f"{obj} are {feature}"


temple_strings = '{} are {}'.format


def temple_strings(obj, feature): 
   return obj+" are "+feature


print(temple_strings("Animals","Good"), 'Animals are Good')
print(temple_strings("Animals","Good"), 'Animals are Good')
print(temple_strings("You","Special"), 'You are Special')
print(temple_strings("lives","frozen"), 'lives are frozen')